src/Blazored.Toast/BlazoredToast.razor.cs

```diff
@@ -42,6 +44,25 @@ protected override async Task OnInitializedAsync()
     public void Close()
         => ToastsContainer.RemoveToast(ToastId);
 
+    private void MouseOver(MouseEventArgs e)
+    {
+        if (Settings.PauseProgressOnHover)
+        {
+            Settings.ShowProgressBar= false;
+            _countdownTimer?.Pause();
+        }
+    }
+
+    private void MouseOut()
```

Comment:
I think this method name should be more descriptive. Perhaps `AttemptResumeCountdown`.